crates/forge_analyzer/src/definitions.rs

```diff
@@ -787,29 +920,34 @@ impl<'cx> FunctionAnalyzer<'cx> {
         fn is_storage_read(prop: &JsWord) -> bool {
             *prop == *"get" || *prop == *"getSecret" || *prop == *"query"
         }
-
         match *callee {
             [PropPath::Unknown((ref name, ..))] if *name == *"fetch" => Some(Intrinsic::Fetch),
+            // [PropPath::Def(def)] => {}]
```

Comment:
Don't forget to handle the case where it's just a "regular" function call, i.e. with just an identifier. Otherwise, we wouldn't catch this:

```javascript
import { sign } from 'jsonwebtoken';
sign('foo', 'secret');
```

Reply:
Yeah, that would return `false` if it's not an import or if it's imported from a different project.